modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceServiceImpl.java

```diff
@@ -81,6 +81,23 @@ public SegmentsExperience deleteSegmentsExperience(
 			segmentsExperienceId);
 	}
 
+	@Override
+	public SegmentsExperience duplicateSegmentsExperience(
+			Locale locale, long segmentsExperienceId,
+			ServiceContext serviceContext)
+		throws PortalException {
+
+		SegmentsExperience segmentsExperience =
```

Comment:
maybe we can call the add and update_priority method at this level, instead of creating a new method in the LocalService, that doesn't have code on this own.

Reply:
well, I first started creating a method in the service layer I was told that the Service layer methods are supposed to have its corresponding methods in the LocalService layer